src/robusta/core/sinks/robusta/robusta_sink.py

```diff
@@ -556,6 +615,16 @@ def __discovery_watchdog(self):
             time.sleep(DISCOVERY_WATCHDOG_CHECK_SEC)
         logging.warning("Watchdog finished")
 
+    def __discover_namespace_resources(self):
```

Comment:
I think this is not in use, and should be removed
Also there's no `__activ` 
Looks like some some leftover